### ISSUE TITLE
Todos mark_as_done not working

### ISSUE THREAD

**Issue by Mikescops** [jdalrymple/gitbeaker#779]
**Description**

The todos.done() method is not working.

**Steps to reproduce**
Example of code:
```
api.Todos.done({ todoId: id })
                        .then((response) => console.log(response))
                        .catch((error: Error) => console.error(error));
```

**Expected behaviour**
Should work.

**Actual behaviour**
Send back 404 not found.

**Possible fixes**
The request is using DELETE method instead of POST.

I tested the same request by replacing the DELETE by POST in the browser and it works well.

**Comment by jdalrymple** [comment]
Ill fix it within the next day! Thanks for pointing it out

**Comment by Mikescops** [comment]
Just a screenshot and I opened a PR above

![image](https://user-images.githubusercontent.com/4266283/81607649-4d27ea00-93d5-11ea-918f-d994a8d2e612.png)

**Comment by Mikescops** [comment]
Can you ping me when the new release will be available? Thanks!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`19.5.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/19.5.0) :rocket:

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`19.5.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/19.5.0) :rocket:

**Comment by Mikescops** [comment]
Thanks a lot! :+1:
